webpack.conf.js

```diff
@@ -54,17 +55,17 @@ function getWebpackConfig({ version, filename, modulename = MODULE_NAME, target
             pathinfo: false
         },
         bail: true,
-        devtool: 'source-map',
         resolve: {
-            extensions: [ '', '.js', '.jsx' ],
+            extensions: [ '.js', '.jsx' ]
         },
         plugins: [
-            new webpack.optimize.UglifyJsPlugin({
+            new webpack.SourceMapDevToolPlugin({
+                filename: '[file].map'
+            }),
+            new UglifyJSPlugin({
                 test: /\.js$/,
-                beautify: !minify,
                 minimize: minify,
-                compress: minify ? { warnings: false } : false,
```

Comment:
Can you make sure it's not spitting out warnings after doing `gulp webpack-major-min`? Just added this setting to disable those warnings.